eng/Microsoft.Extensions.targets

```diff
@@ -8,8 +8,7 @@
    <!-- Microsoft.Maui.Graphics related Packages -->
   <PropertyGroup>
     <_MicrosoftGraphicsWin2DVersion>0.8.0.21</_MicrosoftGraphicsWin2DVersion>
```

Comment:
Is not the Win2D library using a -rc.1.xxx in the version?, should be?

Reply:
That's not available yet, BUT maybe this should use `0.8.2.23`?

Reply:
https://github.com/dotnet/Microsoft.Maui.Graphics/pull/162